test/e2e/config/e2e_conf.yaml

```diff
@@ -153,7 +153,18 @@ providers:
       targetName: "cluster-template-ubuntu.yaml"
     - sourcePath: "../_out/v1.10/cluster-template-upgrade-workload.yaml"
       targetName: "cluster-template-upgrade-workload.yaml"
-  - name: v1.11.99
+  - name: "{go://github.com/metal3-io/cluster-api-provider-metal3@v1.11}"
+    value: "https://github.com/metal3-io/cluster-api-provider-metal3/releases/download/{go://github.com/metal3-io/cluster-api-provider-metal3@v1.11}/infrastructure-components.yaml"
+    type: "url"
+    contract: v1beta1
+    files:
+    - sourcePath: "../data/shared/infrastructure-metal3/v1.11/metadata.yaml"
```

Comment:
you also need to create `/data/shared/infrastructure-metal3/v1.11/metadata.yaml `

Reply:
Thanks done.